interp/type.go

```diff
@@ -885,7 +888,13 @@ func isComplete(t *itype, visited map[string]bool) bool {
 		visited[name] = true
 	}
 	switch t.cat {
-	case aliasT, arrayT, chanT, chanRecvT, chanSendT, ptrT:
+	case aliasT:
+		if t.val != nil && t.val.incomplete && t.val.cat != nilT {
```

Comment:
If the aliased type is already complete, we also want to return true, right?
```suggestion
		if t.val != nil && t.val.cat != nilT {
```